services/report/languages/clover.py

```diff
@@ -14,11 +15,12 @@
 
 
 class CloverProcessor(BaseLanguageProcessor):
-    def matches_content(self, content, first_line, name):
-        return bool(content.tag == "coverage" and content.attrib.get("generated"))
+    def matches_content(self, content: Element, first_line: str, name: str) -> bool:
+        return content.tag == "coverage" and "generated" in content.attrib
```

Comment:
I think there's a slight difference here if `attrib == {"generated": None}`, if that is possible

Reply:
very good point, As these are XML tags/attributes, I’m not sure if their implementation of `in` is equivalent to dicts anyway, I will revert these cases.